portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,19 +43,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(convertToJson(messages));
   }
 
+  @Override
```

Comment:
Can you add function-level javadoc comment for this method? https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Reply:
Done!